code/modules/cargo/packs.dm

```diff
@@ -205,7 +205,7 @@
 
 /datum/supply_pack/security/laser
 	name = "Lasers Crate"
-	cost = 1500
+	cost = 3500
```

Comment:
I don't think we really need to punish cargo like this. Punish the players who abuse it. Having affordable emergency equipment is often times necessary.